scripts/world_bank/health_nutrition_population/population.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+"""world_bank_population.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1f5j8kacwOd_dZqxoO_XCtwuk2jck67bz
+    
+final csv file:
+https://drive.google.com/file/d/1YY8ewSuZD-wx9qJNdvjmz9MB0qJxP-7L/view?usp=sharing&resourcekey=0-ncNblwybOhMGrC2IFPDf_w
+"""
+
+import requests
+import pandas as pd
+
+AGES = 26
+
+series  = [f"SP.POP.AG{age:02d}.{gender}.IN" for age in range(AGES)
+           for gender in ['FE', 'MA']]
+
+def get_df(serieses):
+    '''
+    gets df by iteratively running code for each country + series
+    '''
+    df2 = pd.DataFrame
+    df2 = pd.DataFrame(columns = ['Series Name', 'Series Code', 'Country Code', 'Country',
+                                  'Year', 'Value'])
+    current_series = "&".join(serieses)
+    MAX_PER_PAGE = 32767
```

Comment:
where did this number come from? please leave a comment explaining it.

Reply:
This is the maximum number of rows the api can display, I added a comment above it